README.md

```diff
@@ -63,7 +63,8 @@ None.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| texmf_buffer_size | The value to use for the texmf buffer size. | `200000` | No |
+| texmf_buffer_size | The value to use for the texmf buffer size. | n/a | No |
+| texmf_main_memory | The value to use for the texmf main memory size. | n/a | No |
```

Comment:
Can we set the default value to "Omitted" with a suitable like, like I do [here](https://github.com/cisagov/ansible-role-assessment-tool#role-variables)?

Reply:
If the variable is unspecified we default to `false` as a necessity because of its use in a conditional. I'll try to figure out some way to describe that in brief. I may have to resort to an explanatory few sentences before the table to reasonably convey what's going on since linking to the [documentation about providing default values](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.html#providing-default-values) is not particularly helpful.

Reply:
@jsf9k Please see https://github.com/cisagov/ansible-role-cyhy-reports/pull/70/commits/50b4cd9ada423427026d0cd64f41e52ff14feddd and let me know what you think.